pytorch_to_returnn/torch/nn/functional.py

```diff
@@ -37,6 +37,10 @@ def ones(*size, out=None, dtype=None, layout=None, device=None, requires_grad=Fa
   return tensor(numpy.ones(size, dtype=dtype))
 
 
+def full(size, fill_value, *, out=None, dtype=None, layout=None, device=None, requires_grad=False) -> Tensor:
+  return Tensor(*size, dtype=dtype) + fill_value
```

Comment:
Maybe better using `zeros`.
```suggestion
  return zeros(*size, dtype=dtype) + fill_value
```